src/virtual_elements.js

```diff
@@ -118,7 +118,7 @@ var hasChangedAttrs = function(unused1, unused2, unused3, var_args) {
   }
 
   if (attrsChanged) {
```

Comment:
From what I remember, it slows things down a bit since you need to do the equals check even after you found the first changed attribute.